extensions/amp-ad/0.1/amp-ad-api-handler.js

```diff
@@ -162,6 +170,7 @@ export class AmpAdApiHandler {
     //TODO: make noContentCallback_ default
     if (this.noContentCallback_) {
       this.noContentCallback_();
+      this.noContentCallback_ = null;
```

Comment:
This should probably be part of `#cleanup`, meaning we'd have to move the cleanup call till after this block.